doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -5,25 +5,26 @@ API Changelog
     :local:
     :depth: 1
 
-v6.1
-----
+6.2
```

Comment:
I put these "v"s in here for a reason. The anchor links don't look right otherwise.

Reply:
The file was just updated with the latest version and changes 😄